mhvtl-kmod.spec.in

```diff
@@ -5,7 +5,7 @@
 %global kmodtool /usr/lib/rpm/redhat/kmodtool
 
 # If kversion isn't defined on the rpmbuild line, define it here.
-%{!?kversion: %define kversion 2.6.32-573.12.1.el6.%{_target_cpu}}
+%{!?kversion: %define kversion %(rpm -q kernel --provides | awk '/kernel-uname-r/ {print $3}')}
```

Comment:
You still need to pull the arch off of it:

```
Wrote: /src/rpmbuild/RPMS/x86_64/kmod-mhvtl-1.5-6.2.6.32_696.20.1.el6.x86_64.x86_64.rpm
```

Reply:
I think your comment in Slack about our release abuse is very insightful. We are trying to shove the kversion in there, but that is Old Thinking. Modern kmod RPMs don't do that and neither do we with our newer kmod projects.

I think dropping kversion from release is a great idea.

Reply:
Sounds good- I attempted to use more of kmodtool's magic macros to get a more consistent naming scheme, but that had its own problems and probably isn't worth drilling into more... so, release version changed! (It's also consistent with the -utils spec and keeps a 'dist' flag at the end, i.e. 'el7')